source/blog/2017-12-13-compose-custom-ostree.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Composing custom OSTree update for Fedora 27 Atomic Host
+author: sinnykumari
+date: 2017-12-13
```

Comment:
should this have been 12/13? today is the 14th.

Reply:
Had changed it to a day earlier since I thought we're merging yesterday and it won't get published if it's a day in the future - then left it unmerged in case someone has something to add and contacted @sinnykumari about it just to make sure.